src/platform_impl/windows/window_state.rs

```diff
@@ -197,6 +197,8 @@ impl WindowFlags {
         if self.contains(WindowFlags::DECORATIONS) {
             style |= WS_CAPTION | WS_MINIMIZEBOX | WS_BORDER;
             style_ex = WS_EX_WINDOWEDGE;
+        } else {
+            style |= WS_CAPTION | WS_MAXIMIZEBOX | WS_MINIMIZEBOX | WS_SIZEBOX;
```

Comment:
Do you absolutely need `WS_MINIMIZEBOX` here?

Reply:
Yeah it is needed, Otherwise the window can't be minimized using <kbd>Win</kbd> + <kbd>Arrow Down</kbd>